config/initializers/maxmind.rb

```diff
@@ -1 +1 @@
-DownloadAndExtractMaxmindFileJob.new.extract
+DownloadAndExtractMaxmindFileJob.new.perform
```

Comment:
Thinking out loud here, but we'd see faster boot times if we simply let this run as a background job. Trying to think through the implications of that. 🤔

Reply:
Maybe we only run it synchronously in production. Thoughts?